### DISCUSSION TITLE
Model Output Discrepancies Between Jupyter and VSCode

### DISCUSSION THREAD

**Discussion by FarihaAnis** [microsoft/vscode-discussions#1199] [Extension Development QnA] [answered]
Hi, I've been saving my trained model in JSON format in Jupyter Notebook using Python, assuming it would save all the parameters too. But when I load this model in VSCode, also using Python, the outputs are a bit different, even though both environments are using the same versions of the libraries. Just FYI, I also saved the scaler that I used during training and moved it over to VSCode along with the model using pickle. Any idea why this might be happening? 
Attached is an example how I code in JupterNotebook to save the model and its scaler. 
Thanks!
![image](https://github.com/microsoft/vscode-discussions/assets/143730968/d55d0039-6557-43fd-9dc7-f0112f4a46ee)

**Comment by gjsjohnmurray** [comment, marked as answer]
Crossposted at https://github.com/microsoft/vscode/issues/213437 which is a more appropriate place because this one currently focuses on extension developers.
